### ISSUE TITLE
Custom Page: Video Embed

### ISSUE THREAD

**Issue by glopr** [Xtraball/Siberian#411]
If I insert a video embed script in custom page, the video cannot be set to full screen. Another issue is that after insertig this type of code via source code feature in the editor, after saving, the code isn't editable again because it disappear.

**Comment by Slicit** [comment]
In Custom Page you must use the "Video" Element and not an embed iframe inside the "Text" block, if you need embed videos with iframe you must use the HTML/Source Code feature.

**Comment by glopr** [comment]
Yes, I can video element for standard hosted videos but not with live streaming videos. And yes, I'm using source code feature but after saving the code, if I want to edit it again, the code isn't visible anymore... seems to be deleted or removed. If I don't try to edit the source code, the video still working on app front end until I go to edit source code.
